app/services/gtfs_graph.rb

```diff
@@ -306,98 +321,15 @@ def create_change_payloads(changeset, entity_type, entities)
             changes: changes
           }
         )
-      rescue Exception => e
+      rescue StandardError => e
         log "Error: #{e.message}"
-        log "Payload:"
+        log 'Payload:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.